internal/cli/atlas/integrations/describe_test.go

```diff
@@ -59,9 +59,7 @@ func TestDescribe_Run(t *testing.T) {
 	if err := describeOpts.Run(); err != nil {
 		t.Fatalf("Run() unexpected error: %v", err)
 	}
-	assert.Equal(t, `TYPE    API TOKEN   TEAM       CHANNEL
-SLACK   testToken   testTeam   testChannel
-`, buf.String())
+	snaps.MatchSnapshot(t ,buf.String())
```

Comment:
My preferred solution. It doesn't change much but we no longer need to deal with strings and have reliable way to view differences without dealing with different operating system line breaks or tabs

Reply:
>It doesn't change much but we no longer need to deal with strings and have reliable way to view differences without dealing with different operating system line breaks or tabs

This wasn't a problem because it never went out to the os, it all stayed n memory in the string buffer